parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -106,6 +106,7 @@ write_files:
     KUBELET_CONFIG={{GetKubeletConfigKeyVals .KubernetesConfig }}
     KUBELET_IMAGE={{WrapAsVariable "kubernetesHyperkubeSpec"}}
     DOCKER_OPTS=
+    KUBELET_OPTS=
```

Comment:
Is `KUBELET_OPTS` doing anything here?

Reply:
Yes it is... it is being set when the runtime is set, unfortunately I was going to do this via the config file but the behavior of clear containers for how I usually setup clusters is the system nodes use docker and don't have nested virt enabled, and the agents all have nested virt so it sets `--container-runtime=remote` for only those nodes and not all of them

Reply:
Ah, sorry, didn't see it because gh was truncating the diff with all the whitespace fun! Whitespace has been made less fun, now I can see the `setKubeletOpts` function without clicking things.